setpass/api.py

```diff
@@ -166,5 +168,46 @@ def add(user_id):
     return Response(response=user.token, status=200)
 
 
+@wsgi.app.route('/reset', methods=['GET'])
+def view_reset_form():
+    return render_template('reset_form.html')
+
```

Comment:
I prefer them separate. The route might be the same but the controller function is different.